config/nginx/public.conf

```diff
@@ -85,8 +85,8 @@ http {
     
     ssl_certificate     {{ app_data }}/syncloud.crt;
     ssl_certificate_key {{ app_data }}/syncloud.key;
-    ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
-    ssl_ciphers         HIGH:!aNULL:!MD5;
+    ssl_protocols       TLSv1.2 TLSv1.3;
+    ssl_ciphers         ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
```

Comment:
It permits to avoid use of very old cipher suites that have been cracked a long time ago or just are not efficient about computing time. 

Because before, we used to say to the system "use the best cipher suite possible, else use others that are not safe" now we say "use these suites, else reject the connection" but most of the Internet browsers or devices we use commonly use these suite so there's no compatibility problem and a major security improvement